packages/components/src/Slider/Slider.component.js

```diff
@@ -47,6 +47,29 @@ export function getCaptionsValue(captionsLength, min, max) {
 	return captionsValue;
 }
 
+/**
+ * Return correct modifier styles for usage mode
+ * @param {string} mode
+ * @param {number} value
+ */
+export function getModeStyles(mode, value) {
```

Comment:
Wow, can you simplify this? And use contants like `SLIDER_MODE`

Reply:
We'l remove this and just check `mode` inline with classnames.